libenkf/src/model_config.c

```diff
@@ -634,3 +642,215 @@ void model_config_fprintf_config( const model_config_type * model_config , int e
   fprintf(stream , "\n\n");
 
 }
+
```

Comment:
Hmmm - there was already `model_config` structure, seems that has changed implementation and principles quite dramatically?